phabricatoremails/render/templates/text/public/pinged.text.jinja2

```diff
@@ -0,0 +1,14 @@
+{% import '_macros.text.jinja2' as macros -%}
+
+{{ macros.secure_revision_info(revision) }}
+
+{{ actor_name }} mentioned you.
```

Comment:
Is `pinged` a defined term in Phabricator? I think `mention` is more appropriate, perhaps.

Reply:
I think that "mention" is a good user-facing term, but I think that "ping" is better for technical naming.

We only send users "ping" emails when someone writes their username with an `@` in front. If a first name/last name/full name/nick name is _mentioned_, the user doesn't get an email. It's _only_ when they're "pinged".

That's my vague thoughts on why I landed on the naming I did, but I'm not too attached to it.